index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mobile</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div class="header_logo">My Logo</div>
+            <div class="bar"><i class="fa fa-bars"></i></div>
+        </header>
+        <div class="rectangle-4"><img src="images/Rectangle 4.png" alt=""></div>
+        <div class="ellipse-11"><img src="images/Ellipse 11.png" alt=""></div>
+        <div class="rectangle-7"><img src="images/Rectangle 7.png" alt=""></div>
+        <div class="ellipse-5"><img src="images/Ellipse 5.png" alt=""></div>
+        <div class="ellipse-3"><img src="images/Ellipse 3.png" alt=""></div>
+
+        <div class="rectangle-5"><img src="images/Rectangle 5.png" alt=""></div>
+        <div class="ellipse-4"><img src="images/Ellipse 4.png" alt=""></div>
+        <div class="rectangle-7"><img src="images/Rectangle 7.png" alt=""></div>
+        <div class="ellipse-12"><img src="images/Ellipse 12.png" alt=""></div>
+        <div class="ellipse-13"><img src="images/Ellipse 13.png" alt=""></div>
+        <div class="ellipse-15"><img src="images/Ellipse 15.png" alt=""></div>
+
+        <section class="head-bar">
+            <article>
+                <div class="intro">
+                    I'm Girma Gald to see you!   
+                </div>
+                <div class="discription">
+                    <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! if you like what you  see and have a project you need coded, don't hestiate to contact me</p>
+                </div>
+                <div class="let-connect">
+                    <div class="connect-l">LET'S CONNECT</div>
+                    <div class="icon-i">
+                        <ul>
+                            <li><i class="fa fa-twitter"></i></li>
+                            <li><i class="fa fa-linkedin"></i></li>
+                            <li><i class="fa fa-medium"></i></li>
+                            <li><i class="fa fa-github"></i></li>
+                            <li><i class="fa fa-hand-peace-o"></i></li>
+                        </ul>
```

Comment:
- [ ] Great job on putting these links in lists. However, they are supposed to be links that redirect visitors to respective social media profiles so that people can reach out to you. Please wrap each icon into n anchor tag <a> but also make sure to styles them as they are on the design 😉